src/server/pkg/deploy/assets/assets.go

```diff
@@ -80,6 +81,16 @@ func PachdRc(shards uint64, backend backend) *api.ReplicationController {
 	var backendEnvVar string
 	switch backend {
 	case localBackend:
+		if hostPath == "" {
+			dir, err := ioutil.TempDir("/tmp", "pach-")
```

Comment:
Does this code address that? All this will do is guarantee that the machine that generates the manifest has that directory. The machine that generates the manifest is seldom the machine that runs pachd since most people use the statically generated manifests in the repo. I think for us to use a hostpath in k8s we have to be assuming that k8s will create the directory if it doesn't exist.